core/rawdb/schema.go

```diff
@@ -41,6 +41,9 @@ var (
 	// fastTrieProgressKey tracks the number of trie entries imported during fast sync.
 	fastTrieProgressKey = []byte("TrieSync")
 
+	// oldestIndexedBlockKey tracks the oldest block whose transaction indices(txlookup) has been indexed.
+	oldestIndexedBlockKey = []byte("OldestIndexedBlock")
```

Comment:
I think it's a bit misleading to call this "indexed block", because it's too generic. Let's call it (and update the methods associated) `TxIndexTail`. This way it's obvious both what it is as well as that it's only about txs, not generic block indexing.